apps/dashboard/app/models/project.rb

```diff
@@ -199,6 +199,27 @@ def jobs
     end.flatten
   end
 
+  def readme_path
+    md_readme = "#{directory}/README.md"
+    return md_readme if File.exist?(md_readme) && File.readable?(md_readme)
+
+    txt_readme = "#{directory}/README.txt"
+    return txt_readme if File.exist?(txt_readme) && File.readable?(txt_readme)
```

Comment:
I think we can use globs to simplify this. Although, I think methods calling `readme_path` have to account for `nil` values, but it seems like they would need to already?

```
    file = Dir.glob("#{directory}/README.{md,txt}").first.to_s
    File.readable?(file) ? file : nil
```